source/content/guides/decoupled-sites/03-starter-kits.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Pantheon Decoupled
+subtitle: Starter Kits
+description: Use a Pantheon starter kit to congfire a decoupled site backend.
+categories: [platform]
+tags: [webops, workflow, decoupled]
+contributors: [joa-pan]
+type: guide
+layout: guide
+showtoc: true
+anchorid: starter-kits
+permalink: docs/guides/decoupled-sites/starter-kits/
+editpath: decoupled-sites/03-starter-kits.md
+reviewed: "2022-07-31"
+---
+
+## Creating a New CMS Backend for a Decoupled Frontend
+
+You can create a decoupled site using a Pantheon starter kit. Choose from either a WordPress or Drupal CMS. You can also create a site without any CMS. 
+
+
+## Installation
+
+<TabList>
+
+<Tab title="Drupal Backend Installation" id="drupal-install" active={true}>
+
+- Run the `terminus build:project:create` as follows:
+
+  ```
+  terminus build:project:create \
+    --team='My Team Name}' \
+    --template-repository="git@github.com:pantheon-systems/decoupled-drupal-recommended.git" pantheon-systems/decoupled-drupal-recommended \
+    --ci-template='git@github.com:pantheon-systems/advanced-ci-templates' \
+    --visibility private {PROJECT_NAME} \
+    --stability=dev \
+    --profile="pantheon_decoupled_profile"
+  ```
+
+  * Replace `<{PROJECT_NAME}>` with your project name, for example `decoupled-drupal`.
+
+  * Replace `<'{My Team Name}'>` with your team name, for example `My Agency`. This can also be omitted.
+
+**Note:** This will result in a GitHub repository created for this new codebase under the authenticated user's namespace (unless the `--org` option is used), a site created on Pantheon and a CircleCI project created for automated deployments.
+
+</Tab>
+
+<Tab title="WordPress Backend Installation" id="wordpress-install">
+
+- Run the `terminus build:project:create` as follows:
+
+  ```
+  terminus build:project:create \
+    --team='{My Team Name}' \
+    --template-repository="git@github.com:pantheon-systems/decoupled-wordpress-recommended.git" \
+    pantheon-systems/decoupled-wordpress-recommended \
+    --ci-template='git@github.com:pantheon-systems/advanced-ci-templates' \
+    --visibility private {PROJECT_NAME} \
+    --stability=dev
+  ```
+
+  * Replace `{PROJECT_NAME}` with a Project name for example `decoupled-wordpress`.
+
+  * Replace `{My Team Name}` with your team name - for example `My Agency`. This can also be omitted.
+
+**Note:** This will result in a GitHub repository created for this new codebase, a site created on Pantheon and a CircleCI project created for automated deployments.
+
+</Tab>
+</TabList>
+
```

Comment:
Include Import Repo backend config:

There is no code to run. They just need a  GitHub repo with content in it and then will link the CMS in the Pantheon decoupled dashboard. - Should we mention that we support this , because this might not supposed to be exposed. 

*_When you link the CMS it creates an environment variable that your front end will need to know about._  -